.circleci/config.yml

```diff
@@ -47,6 +48,9 @@ parameters:
   pachd-latest-version:
     type: string
     default: ""
+  go-version:
+    type: string
+    default: "1.19"
```

Comment:
what is this for?

Reply:
It's for line 156 in config.yaml. But, the real reason is that `make install` requires go 1.19 so for the fallback in run_api_performance_test.sh we need go 1.19+ and the image comes with 1.18.